### ISSUE TITLE
[Outreachy Task Submission] A missing form label

### ISSUE THREAD

**Issue by Echika** [ushahidi/platform#4803]
The 'SEARCH POST BY KEYWORD' button has a missing form label.  If a form control does not have a properly associated text label, the purpose of that form control may not be presented to screen reader users. Form labels  should also provide visible descriptions and larger clickable targets for form controls.

**Comment by RashJrEdmund** [comment]
hi @Echika, good morning...
you've created 2 issues that are similar.

I think you should close the previous one you created since it's the same here.
